source/common/stats/isolated_store_impl.h

```diff
@@ -234,10 +235,12 @@ class IsolatedStoreImpl : public StoreImpl {
 
   void forEachScope(SizeFn f_size, StatFn<const Scope> f_stat) const override {
     if (f_size != nullptr) {
-      f_size(1);
+      f_size(scopes_.size() + 1);
```

Comment:
This is unrelated to this change but because of this change we can now do the API correctly? Is that right?

Reply:
Sorry, which API are you referring to?

Reply:
`forEachScope`